### ISSUE TITLE
Tooltip not allowing children elements

### ISSUE THREAD

**Issue by albert-truepill** [react-native-elements/react-native-elements#3668]
### Screenshot

![Screen Shot 2022-10-27 at 11 13 43 AM](https://user-images.githubusercontent.com/81997575/198367000-faa18036-4d23-4ef9-b5ba-a09513296438.png)
(This is what happens when I copy+paste the example code: https://reactnativeelements.com/docs/components/tooltip#usage)

### Explain the problem

VSCode is throwing this typescript error when trying to use `Tooltip`:

```
No overload matches this call.
  Overload 1 of 2, '(props: TooltipProps, context?: any): ReactElement<any, any> | null', gave the following error.
    Type '{ children: Element; popover: Element; }' is not assignable to type 'IntrinsicAttributes & TooltipProps'.
      Property 'children' does not exist on type 'IntrinsicAttributes & TooltipProps'.
  Overload 2 of 2, '(props: TooltipProps): ReactElement<any, any> | null', gave the following error.
    Type '{ children: Element; popover: Element; }' is not assignable to type 'IntrinsicAttributes & TooltipProps'.
      Property 'children' does not exist on type 'IntrinsicAttributes & TooltipProps'.ts(2769)
```

My code:
```
import { Icon, Text, Tooltip } from '@rneui/themed'

...

<Tooltip popover={<Text>{'test'}</Text>}>
  <Text>{'Tooltip here'}</Text>
</Tooltip>
```

FWIW, anything I put in between Tooltip causes this error message.

I also tried creating a new file `Tooltip.tsx` and used the example code [here](https://reactnativeelements.com/docs/components/tooltip#usage) and it also gives the same error.

### Environment
VSCode 
- Version: 1.72.2 
- Date: 2022-10-12T22:16:30.254Z

Dependencies:
```
"react-native": "0.70.3"
"react-native-elements": "3.4.2"
"@rneui/themed": "^4.0.0-rc.6"
"@types/react": "^18.0.21",
"@types/react-native": "^0.70.4",
"typescript": "^4.8.3"
```

### Steps To Reproduce

1. Open VSCode + React-Native project

2. Create a new file and copy+paste the example code https://reactnativeelements.com/docs/components/tooltip#usage:

```
import React from 'react';
import { Tooltip, Text, lightColors } from '@rneui/themed';
import { View, StyleSheet, Dimensions, ScrollView } from 'react-native';

const { height } = Dimensions.get('window');

type ToolTipComponentProps = {};

const TooltipComponent: React.FunctionComponent<ToolTipComponentProps> = () => {
const toolProps = {};
return (
  <>
    <ScrollView>
      <View style={{ marginVertical: height / 8 }}>
        <View style={styles.view}>
          <Tooltip
            popover={<Text>no caret!</Text>}
            withPointer={false}
          >
            <Text>without caret</Text>
          </Tooltip>
          <Tooltip
            popover={<Text>Tooltip info goes here</Text>}
            width={200}
            backgroundColor={lightColors.primary}
          >
            <Text>Press me</Text>
          </Tooltip>
        </View>
        <View style={styles.view}>
          <Tooltip
            backgroundColor={lightColors.secondary}
            popover={
              <Text>Tooltip info goes here too. Find tooltip everywhere</Text>
            }
            containerStyle={{ width: 200, height: 60 }}
          >
            <Text>Press me</Text>
          </Tooltip>
          <Tooltip
            containerStyle={{ width: 145, height: 130 }}
            popover={
              <Text>
                {
                  'Some big text full of important stuff for the super duper user that our design has been created for'
                }
              </Text>
            }
          >
            <Text>HUGE</Text>
          </Tooltip>
        </View>
        <View style={styles.view}>
          <Tooltip
            width={200}
            backgroundColor={lightColors.primary1}
            popover={<Text>Tooltip info goes here</Text>}
          >
            <Text>More attention</Text>
          </Tooltip>
        </View>
        <View style={styles.view}>
          <Tooltip
            width={200}
            backgroundColor={lightColors.primary2}
            popover={<Text>Tooltip info goes here</Text>}
          >
            <Text>I'm different</Text>
          </Tooltip>
          <Tooltip
            width={200}
            popover={<Text>Tooltip info goes here</Text>}
          >
            <Text>Press me</Text>
          </Tooltip>
        </View>
      </View>
    </ScrollView>
  </>
);
};

const styles = StyleSheet.create({
view: {
  display: 'flex',
  flexDirection: 'row',
  justifyContent: 'space-evenly',
  marginVertical: 50,
},
});

export default TooltipComponent;
```

3. See error

**Comment by Nantris** [comment]
I wonder if this is the cause of #3667

**Comment by arpitBhalla** [comment]
Fixed by #3663
